daq/runner.py

```diff
@@ -269,7 +269,9 @@ def main_loop(self):
                 self._flush_faucet_events()
             LOGGER.info('Entering main event loop.')
             LOGGER.info('See docs/troubleshooting.md if this blocks for more than a few minutes.')
-            self.stream_monitor.event_loop()
+            while self.stream_monitor.event_loop():
+                for host in list(self.mac_targets.values()):
```

Comment:
_might_ consider extracting this to a (trivial) function (self.check_module_timeouts() -- plural) -- just because logically it's weird to expose mac_targets here).